### PR TITLE
Updating Readme

### PR DESCRIPTION
**Pull request by ghost**
Removed libtoxcore.so link and linked to new Jenkins build; made the whole thing more like markdown (instead of the HTML-like file it was before); changed short links at the top to dots and added download notes which address https://github.com/tux3/qTox/issues/2085 a bit (explain what you download & update note)

### CODE REVIEW

**Comment by zetok** [comment]
Generally, that's not something that could be merged.

The part that could be merged is:

```
Below you can download the latest nighly builds. As they do not update automatically at this stage of development make sure you get back to this site and regularely download the latest version of qTox.
```

Preferably in a clean PR, in a single commit.

# 

more ≠ better

**Comment by ghost** [comment]
Okay so that's the part which can be merged. What are the parts which you do not like and why?

**Comment by antis81** [comment]
@blobgo Please correct comment.

**Comment by zetok** [comment]
> Okay so that's the part which can be merged. What are the parts which you do not like and why?

Aside from content, there are used 3 commits for a really small changeset in 1 file - that's not quite how things should be done `:)`.

About rest of stuff, I'll comment on per-change basis.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 qTox
 ======
 
- [**Compiling**](/INSTALL.md) **|** [**Contributing**](https://github.com/tux3/qTox/wiki#contributing) **|** [**Reporting bugs**](https://github.com/tux3/qTox/wiki/Writing-Useful-Bug-Reports) **|** [**Translating**](https://github.com/tux3/qTox/wiki/Translating) **|** [**Mac & Linux jenkins**](https://jenkins.libtoxcore.so/user/tux3/my-views/view/qTox/) **|** [**Keyboard shortcuts**](https://github.com/tux3/qTox/wiki/Keyboard-shortcuts) **|** [**Mailing list**](https://lists.tox.chat) **|** **IRC Channel:** [#tox@freenode](https://webchat.freenode.net/?channels=tox)
+ [**Compiling**](/INSTALL.md) **⦁** [**Contributing**](https://github.com/tux3/qTox/wiki#contributing) **⦁** [**Reporting bugs**](https://github.com/tux3/qTox/wiki/Writing-Useful-Bug-Reports) **⦁** [**Translating**](https://github.com/tux3/qTox/wiki/Translating) **⦁** [**Jenkins builds**](https://build.tox.chat/) **⦁** [**Keyboard shortcuts**](https://github.com/tux3/qTox/wiki/Keyboard-shortcuts) **⦁** [**Mailing list**](https://lists.tox.chat) **⦁** **IRC Channel:** [#tox@freenode](https://webchat.freenode.net/?channels=tox)
```
Comment (zetok) [inline README.md]: First, change from `|` to `⦁` - doesn't seem to look good.

As for replacing link to jenkins - it's good.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
-====
+qTox is a powerful Tox client that follows the Tox design guidelines while running on all major platforms.
 
-<h5>qTox is a powerful Tox client that follows the Tox design guidelines while running on all major platforms:</h5>
```
Comment (zetok) [inline README.md:8]: That text size was chosen and agreed on - change doesn't look better.

File: README.md [hunk 3]
```diff
@@ -18,13 +21,15 @@ qTox
...
-<h4>Screenshots</h4>
-Note: The screenshots may not always be up to date, but they should give a good idea of the general look and features
-<img src="http://i.imgur.com/hwGmDeK.png"/>
-<img src="http://i.imgur.com/tmX8z9s.png"/>
+## Screenshots
+**Note:** The screenshots may not always be up to date, but they should give a good idea of the general look and features.
+
+
+![Screenshot 01](https://i.imgur.com/hwGmDeK.png)
+![Screenshot 02](https://i.imgur.com/tmX8z9s.png)
```
Comment (zetok) [inline README.md:28]: Again, text size was carefully picked, and this change breaks it.

As for bold `Note` - it also doesn't look good.

The only change here that could be applied is markdown-style for screenshots.. although, if screenshots "work" well, then does it really need changing?

File: README.md [hunk 4]
```diff
@@ -18,13 +21,15 @@ qTox
...
 
 
-<h4>Features</h4>
+## Features
```
Comment (zetok) [inline README.md]: And this one also breaks carefully picked text size.

File: README.md (hunk 1 above)
Comment (zetok) [inline README.md]: Actually, upon pondering about this change a bit longer, I see that while dots aren't the best, due to them being widechars they make links appear to be less cluttered - that's nice, and that too will be good to merge. `:)`

Comment (antis81) [inline README.md]: While at it, can you also correct headers to present time, please? It should be "Contribute", not "Contributing" etc.

**Comment by suhr** [comment]
There's so much _carefully picked_ sizes...

**Comment by antis81** [comment]
@suhr Maybe, but make a `<h2>` out of an `<h4>` is not the right thing to do here. Instead, four hashes would be ok -> like this:

#### This is a h4 header :)

**Comment by zetok** [comment]
@suhr sorry, but your irony is a bit lost on me, since I had to figure out exact sizes that would look good, and at the same time satisfy @tux3's tastes: #1767

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md]: @zetok 
So what should I revert it to | or use dots? Also a | with a few more spaces around would be possible.

Comment (ghost) [inline README.md]: > While at it, can you also correct headers to present time, please? It should be "Contribute", not "Contributing" etc.

Done.

Comment (antis81) [inline README.md]: @blobgo :+1:

**Comment by ghost** [comment]
I also don't understand how a text size can be such important, but anyway I made smaller.
I also changed to present tense and removed the bold **Note:** although I liked it.
But well... that's all subjective. So I hope that's better now.

I can't see the problem with more of one commit. That's just how I added it...

File: README.md (hunk 2 above)
Comment (antis81) [inline README.md:8]: @blobgo Can you please change that back to an `<h5>` header? You can use hashes like:

##### ##### This is a h5 header

**Comment by antis81** [comment]
@blobgo Try to seperate between a textual correction (like: chose better wording) and a structural fix (like change header 5 to header 1). Obviously you are right, because misusing the header depth to make the document "look more nicely" destroys it's structure. Rather than using those nasty tricks, the style sheet should be changed to match wanted appearance. AFAIK we cannot change the builtin Github style. So there's nothing we can do about it, unless we host our own servers.

File: README.md (hunk 2 above)
Comment (ghost) [inline README.md:8]: No because that's no headline. I can make it bold, which looks practically the same as #####.

**Comment by ghost** [comment]
Come on... don't hyperbolize this. It's just a Markdown file so we don't talk about style sheets here.
About [this](https://github.com/tux3/qTox/pull/2087#discussion_r37694174) I've made it bold now.

And if you want here is an comparison of a h5 header and bold text. So guess what is what:
toxreadmeexample

**Comment by zetok** [comment]
> I also don't understand how a text size can be such important, but anyway I made smaller.
> It's just a Markdown file so we don't talk about style sheets here.

Sadly, it's not _any_ markdown file. If that was the case, proposed by you changes would have been merged long time ago, provided that you would have squashed commits.

Well, what you're changing, is basically qTox's "front page" - as such it's rather important to make things on it look good.
Font size is just one of things that make it look in particular way - and no, "looking good" isn't really subjective, since it's mostly a matter of 
a) exposing required from a front page functionality, in a way most fitting for visitor
b) letting visitor find what they desire the most concentrated in one place, in a clear, not cluttered way
c) there's no need to state obvious things, but rather visitor should be instinctively pointed to sections that they are looking for

With that in mind:

bloat
- `Download` header is completely unnecessary there - you already have `download` as a part of every link. It's an obvious thing, no need to put it there. 
  
  Beside being unnecessary, `Download` there also makes everything below to be pushed 
- Though, if you want to be able to link to `#download` link, to be able to link to download section without unnecessarily bother people with looking at organisation of source code, you could use HTML's `
  <a name="` stuff, or just simply keep `qTox is a powerful (…)` a `<h5>` tag…
- And part `download` in the links stays there, since when someone one is looking at the page, the ~first thing they focus on are colours - links are nicely blue, exposing 32/64 bit **download** of the client they desire the most. I.e. they see link, **bold** OS, and clear **[32/64 download](link)**, all concentrated in single place, in a not cluttered way.
- Also text /below/ `Download` - writing `Below` is not needed, user is going to scroll down a bit anyway, and download links are nicely, intuitively exposed there.
  
  Text probably can be changed:
  `Below you can download the latest nighly builds. As they do not update automatically (…)`
  ↓
  `Nightly builds do not update automatically (…)`
  
  Also, there is a typo there: `regularely` → `regularly`
  
  I'm not sure if it wouldn't be beneficial to just put that text below download links - it's not like putting it on top would make anyone actually read it, and it only makes getting to the relevant download link more bothersome…

# 

> removed the bold Note: although I liked it.

I'm sorry, but that doesn't really matter. Sadly, what matters is "how much does this element matter" ?
Answer to that is.. it doesn't. At all. There's no reason at all to bring viewer attention to this, since they're going to look at images anyway, and not bother reading some boring, unimportant disclaimer. That's why making `Note` bold isn't a good idea.

# 

> I can't see the problem with more of one commit. That's just how I added it...

Number of commits is not a problem. What is a problem is a ratio of commits per change - when it comes to small (in size) changes, keeping number of commits as low as possible is the best way to go - in this particular case, _1_ commit would be just the right amount. 2 commits if someone is not familiar with git workflow, they contribute for the ~first time to qTox, and change is important.

Generally, keeping number of commits low per small changes helps keeping
a) repo size smaller
b) git history is cleaner, since it's not being bloated with unnecessary commits.

File: README.md [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
+ [**Compile**](/INSTALL.md) **⦁** [**Contribute**](https://github.com/tux3/qTox/wiki#contributing) **⦁** [**Report bugs**](https://github.com/tux3/qTox/wiki/Writing-Useful-Bug-Reports) **⦁** [**Translate**](https://github.com/tux3/qTox/wiki/Translating) **⦁** [**Jenkins builds**](https://build.tox.chat/) **⦁** [**Keyboard shortcuts**](https://github.com/tux3/qTox/wiki/Keyboard-shortcuts) **⦁** [**Mailing list**](https://lists.tox.chat) **⦁** **IRC Channel:** [#tox@freenode](https://webchat.freenode.net/?channels=tox)
 
-====
+**qTox is a powerful Tox client that follows the Tox design guidelines while running on all major platforms.**
```
Comment (antis81) [inline README.md:6]: @blobgo Yes, you're right. This is not  a header. :+1:

**Comment by antis81** [comment]
@zetok The essence to this PR boils down to the fixed links in the header. We can rather restructure the text in a new PR. If you want I'll squash changes into a single commit and sign-off locally. Would that be ok?

**Comment by zetok** [comment]
> @zetok The essence to this PR boils down to the fixed links in the header. We can rather restructure the text in a new PR. If you want I'll squash changes into a single commit and sign-off locally. Would that be ok?

Well, even after squashing points I've mentioned about still remain (e.g. `Download` not being needed).

If that gets fixed and squashed, it would be more than just 'ok' `:)`

**Comment by ghost** [comment]
Here you go (one commit): https://github.com/tux3/qTox/pull/2104

**Pull request closed without merging**